core/src/main/java/com/gentics/mesh/core/actions/impl/LanguageDAOActionsImpl.java

```diff
@@ -0,0 +1,93 @@
+package com.gentics.mesh.core.actions.impl;
+
+import static com.gentics.mesh.core.rest.error.Errors.error;
+import static io.netty.handler.codec.http.HttpResponseStatus.BAD_REQUEST;
+
+import java.util.function.Predicate;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+import com.gentics.graphqlfilter.filter.operation.FilterOperation;
+import com.gentics.mesh.context.BulkActionContext;
+import com.gentics.mesh.context.InternalActionContext;
+import com.gentics.mesh.core.action.DAOActionContext;
+import com.gentics.mesh.core.action.LanguageDAOActions;
+import com.gentics.mesh.core.data.HibLanguage;
+import com.gentics.mesh.core.data.page.Page;
+import com.gentics.mesh.core.data.perm.InternalPermission;
+import com.gentics.mesh.core.db.Tx;
+import com.gentics.mesh.core.rest.lang.LanguageResponse;
+import com.gentics.mesh.event.EventQueueBatch;
+import com.gentics.mesh.parameter.PagingParameters;
+
+@Singleton
+public class LanguageDAOActionsImpl implements LanguageDAOActions {
+
+	@Inject
+	public LanguageDAOActionsImpl() {
+	}
+
+	@Override
+	public HibLanguage create(Tx tx, InternalActionContext ac, EventQueueBatch batch, String uuid) {
+		throw error(BAD_REQUEST, "error_language_creation_forbidden");
+	}
+
+	@Override
+	public HibLanguage loadByUuid(DAOActionContext ctx, String uuid, InternalPermission perm, boolean errorIfNotFound) {
+		return ctx.tx().languageDao().findByUuid(uuid);
```

Comment:
This ignores errorIfNotFound, which causes subsequent NPEs when language endpoints are called with incorrect uuid's